Sources/Task/TaskProgress.swift

```diff
@@ -120,21 +178,25 @@ extension NSProgress {
     ///
     /// Send `isOrphaned: false` if the iOS 9 behavior cannot be trusted (i.e.,
     /// `progress` is not understood to have no parent).
```

Comment:
Double negative - so, "i.e., `progress` is understood to have a parent"?

(I don't understand what this really means, or why we'd mistrust the iOS 9 behavior.)

Return type also is not documented, or why it returns and how it differs from the receiver.